backend/routes.js

```diff
@@ -217,7 +217,12 @@ route.POST('/event', {
         const manifest = await sbp('chelonia.db/get', deserializedHEAD.head.manifest)
         const parsedManifest = JSON.parse(manifest)
         const { name } = JSON.parse(parsedManifest.body)
-        if (name !== 'gi.contracts/identity') return Boom.unauthorized('This contract type requires ownership information', 'shelter')
+        if (name !== 'gi.contracts/identity') {
+          return Boom.unauthorized('This contract type requires ownership information', 'shelter')
+        }
+        if (process.env.CHELONIA_REGISTRATION_DISABLED) {
+          return Boom.unauthorized('Registration disabled', 'shelter')
```

Comment:
401 Unauthorized doesn't sound like the right response type, since it's used to indicate the need for credentials, and in this case credentials won't help. 403 Forbidden sounds more apt for this kind of situation, or else a generic 400 Bad Request.

Also, for future reference, this logic here might need to change a bit as the server is made more generic (the issue is `(name !== 'gi.contracts/identity')`, which is GI-specific). I think this is one of the only few places where logic needs to be rethought.